flutter/lib/ui/geometry.dart

```diff
@@ -880,19 +1033,76 @@ class RRect {
   static RRect lerp(RRect a, RRect b, double t) {
     if (a == null && b == null)
```

Comment:
in lerp(), use a private constructor so that you can avoid first creating all the Radius objects